designer/server/src/main/scala/pl/touk/nussknacker/ui/factory/LocalProcessingTypeDataStateFactory.scala

```diff
@@ -22,7 +24,8 @@ class LocalProcessingTypeDataStateFactory(
   override def create(
       designerConfig: ConfigWithUnresolvedVersion,
       getDeploymentManagerDependencies: ProcessingType => DeploymentManagerDependencies,
-      additionalUIConfigProvider: AdditionalUIConfigProvider
+      additionalUIConfigProvider: AdditionalUIConfigProvider,
+      workingDirectoryOpt: Option[Path]
```

Comment:
Why Opt? I haven't seen much use for this kind of standard.

Reply:
`find -name *.scala -exec grep 'Opt: ' {} \; | wc -l` returned 127 entries - it is quite common. I've seen three approaches in scala:
1. using `Opt` suffix
2. using `maybe` prefix
3. skipping Opt/maybe (I assume that this one you propose)
 
1. and 2. approach has the benefit that whey you do map on this field you don't have to think how to call the mapped value (`workingDirectoryOpt.map { workingDirectory=>`). How do you name the mapped values in the proposed approach?

Reply:
So we can make agreement that in the future Option value we will named with postfix Opt? It's fine by me.